packages/react-components/form-elements/src/Switch/Label.tsx

```diff
@@ -0,0 +1,53 @@
+import tokens from '@datacamp/waffles-tokens';
+import { css } from '@emotion/react';
+
+import { SwitchProps } from './index';
+
+const labelStyle = css({
+  alignItems: 'center',
+  boxSizing: 'border-box',
+  color: tokens.colors.navyText,
+  cursor: 'pointer',
+  display: 'inline-flex',
+  fontFamily: tokens.fontFamily.sansSerif,
+  fontSize: 16,
+  fontWeight: parseInt(tokens.fontWeight.regular, 10),
+  opacity: 1,
+  position: 'relative',
+  userSelect: 'none',
+});
+
+type LabelProps = Pick<
+  SwitchProps,
+  'appearance' | 'children' | 'className' | 'disabled'
+>;
+
+function Label({
```

Comment:
Did you test it with a multiline label?

Reply:
Yes, I think it looks pretty good. Now the question is whether we want to align everything to the centre or to the top. I will ask designers on Friday for a fedback.
<img width="457" alt="Screenshot 2021-05-26 at 13 40 09" src="https://user-images.githubusercontent.com/20565536/119653741-fb0f0680-be27-11eb-8902-418bbbf386c6.png">